setup.py

```diff
@@ -12,6 +12,7 @@
         'collective.geo.contentlocations',
         'collective.z3cform.datagridfield < 1.4.0',
         'ftw.shop',
+        'plonetheme.onegov < 4.0.0'
```

Comment:
I suggest to move that into the Plone 4 buildout.
We have deployments with Plone 5, publisher and plonetheme.onegov 4

Reply:
It's in the `tests_plone4` extra and so it will only be used for Plone 4 already.